### PR TITLE
Assignment 1

### PR DESCRIPTION
**Pull request by neethil**
## What changes are you trying to make? (e.g. Adding or removing code, refactoring existing code, adding reports)
labs 1, 2, 3, assignment 1

## What did you learn from the changes you have made?
neural networks, tensorflow, keras

## Was there another approach you were thinking about making? If so, what approach(es) were you thinking of?
no

## Were there any challenges? If so, what issue(s) did you face? How did you overcome it?
no

## How were these changes tested?
in google colab

## Checklist
- [ ] I can confirm that my changes are working as intended

### CODE REVIEW

File: 01_materials/labs/lab_1.ipynb
Comment (tianyi21) [inline 01_materials/labs/lab_1.ipynb:1]: - b: Please always reinitialize your model with `model = Sequential() ...` to reset all model parameters in addition to `model.compile()`. Or, you are continuing the training from the previous trained state with a different model setup. You may check this with the accuracy of the first epoch.

File: 02_activities/assignments/assignment_1.ipynb
Comment (tianyi21) [inline 02_activities/assignments/assignment_1.ipynb:1]: - Task 3c, 4: accuracy won't exceed 1 so it's safe to plot `plt.ylim(0,1)`
- To further improve the model, you may consider adding an additional `Conv2d` layer.

**Review by tianyi21 — APPROVED** [review]
Hi Neethila!

Thank you for submitting your DL assignments. Please refer to my comments attached to each file.

Your A1 and Lab 1-3 are now complete.

Thanks,
Tianyi [LS]
